### PR TITLE
CCD-1580: Integrate Fortify Scan into Nightly Pipelines

### PR DESCRIPTION
**Pull request by rajaram-nagarajan**
### JIRA link  ###

https://tools.hmcts.net/jira/browse/CCD-1580

### Change description ###

Integrated Fortify Scan into Nightly Pipelines

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: config/fortify-client.properties [hunk 1]
```diff
@@ -0,0 +1 @@
+fortify.client.releaseId=65430
```
Comment (MSancaktutar) [inline config/fortify-client.properties:1]: release id is 65417. please pick the respective value for each repo.

**Review by MSancaktutar — CHANGES_REQUESTED** [review]
Please correct the respective release id - otherwise perfect.

File: config/fortify-client.properties (hunk 1 above)
Comment (rajaram-nagarajan) [reply config/fortify-client.properties]: done

**Review by MSancaktutar — APPROVED** [review]
LGTM

**Pull request merged**